_data/projects/gkv.yml

```diff
@@ -0,0 +1,12 @@
+name: Gkv
+desc: An open source, git-based key value store for small, simple projects
+site: https://github.com/ybur-yug/gkv
+tags:
+  - oss
+  - databases
+  - ruby
+  - db
+  - kv
```

Comment:
Perhaps don't worry about these acronyms. 

As you're using `git` here in an interesting way, perhaps call that out.